app/views/reset_passwords/edit.html.slim

```diff
@@ -0,0 +1,29 @@
+.header-logo.header-logo--light
+  a href = t('footer.home_url')
+    .header-logo__logo.sumofus-logo--positive
+.center-content
+  .center-content__one-column.registration
+    .center-content__central-square
+      h1.typography__highlight
+        = t('.title')
+        br
+        = @authentication.member.email
+
+      - if @authentication.errors.any?
+        div.flash-alert
+          - @authentication.errors.full_messages.each do |message|
+            p
+              strong = message
+
+      p = t('.description')
+
+      = form_tag reset_password_path, method: :put, class: 'form--big dark registration-form' do
+        div.form__group.action-form__field-container
```

Comment:
`div` is unneccessary, you can just call `.form__group.action-form__field-container`